DrinkManagerWeb/Views/Drinks/SearchByAlcoholContent.cshtml

```diff
@@ -0,0 +1,35 @@
+﻿@using BLL
+@model DrinkManagerWeb.Models.ViewModels.DrinksViewModel
+
+@{
+    ViewBag.Title = "Drinks";
+    Layout = "_Layout";
+}
+<div class="row px-4">
+    <div class="col-12">
+        <header class="d-flex align-items-center">
+            <h1>Search by Alcohol Content</h1>
+        </header>
+        @*Checkboxes returns user choice of alcohol content to controller. 
+            They are automatically checked when user comes to this view from another view*@
+        <div>
+            <form asp-route-pageNumber="1" method="post">
+                <div class="form-check">
+                    @Html.CheckBoxFor(item => item.Alcoholics, new { @class = "form-check-input", @id = "alcoholicsCheck" })
+                    <label class="form-check-label" for="alcoholicsCheck">Alcoholic Drinks</label>
+                </div>
+                <div class="form-check">
+                    @Html.CheckBoxFor(item => item.NonAlcoholics, new { @class = "form-check-input", @id = "nonAlcoholicsCheck" })
+                    <label class="form-check-label" for="nonAlcoholicsCheck">Non Alcoholic Drinks</label>
+                </div>
+                <div class="form-check">
+                    @Html.CheckBoxFor(item => item.OptionalAlcoholics, new { @class = "form-check-input", @id = "optionalAlcoholicsCheck" })
+                    <label class="form-check-label" for="optionalAlcoholicsCheck">Optional Alcohol Drinks</label>
+                </div>
+                <button>Search</button>
```

Comment:
Could you use the same "Search" button and add "Clear" button as we have in search by ingredients and by name pages?

Reply:
Done!

Reply:
If you add some space between choice options, Search and Clear buttons and showing details, that will be great. :)